front/lib/swr/assistants.ts

```diff
@@ -242,6 +243,32 @@ export function useAgentConfiguration({
   };
 }
 
+export function useAgentConfigurationFeedbacks({
```

Comment:
Do we want any kind of pagination there or do we think it's not worth it for now?
(I think we can wait until someone tells us it's hard to manage)